### PR TITLE
Add sensor platform to Motionblinds BLE

### PR DESCRIPTION
**Pull request by LennP**
## Proposed change

Add sensor platform to Motionblinds BLE

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/114226
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Expanded Motionblinds Bluetooth integration in Home Assistant to include new sensor entities for:
    - Battery percentage
    - Calibration status
    - Connection status
    - Signal strength

### CODE REVIEW

**Comment by c0ffeeca7** [comment]
Could you add the types of sensors that are added and describe the configuration details?
For example, like they did here: https://github.com/home-assistant/home-assistant.io/blob/22c2bfe7b276d468f5761ecdac68ec05f8fd78ca/source/_integrations/traccar_server.markdown?plain=1#L120-L137

**Comment by LennP** [comment]
> Could you add the types of sensors that are added and describe the configuration details? For example, like they did here:
> 
> https://github.com/home-assistant/home-assistant.io/blob/22c2bfe7b276d468f5761ecdac68ec05f8fd78ca/source/_integrations/traccar_server.markdown?plain=1#L120-L137

The current documentation already contains information on this. I think it will be a bit confusing if I include other headers with information about these sensors as they are quite straightforward?

```
## Setup

During the setup of a Motionblinds BLE device, you will be asked what kind of blind your Motionblind is. There are 8 different blind types:

- **Roller blind**: has the ability to change position and speed.
- **Honeycomb blind**: has the ability to change position and speed.
- **Roman blind**: has the ability to change position and speed.
- **Venetian blind**: has the ability to change position, tilt, and speed.
- **Venetian blind (tilt-only)**: has the ability to change tilt and speed.
- **Double Roller blind**: has the ability to change position, tilt, and speed.
- **Curtain blind**: has the ability to change position. May need to be calibrated if the end positions are lost, which can be done by using the open/close cover button or the set cover position slider. This will trigger a calibration which will first make the curtain find the end positions after which it will run to the position as indicated by the command that was given.
- **Vertical blind**: has the ability to change position and tilt. May need to be calibrated if the end positions are lost, which has to be done using the Motionblinds BLE mobile app.
```

**Comment by c0ffeeca7** [comment]
> > Could you add the types of sensors that are added and describe the configuration details? For example, like they did here:
> > https://github.com/home-assistant/home-assistant.io/blob/22c2bfe7b276d468f5761ecdac68ec05f8fd78ca/source/_integrations/traccar_server.markdown?plain=1#L120-L137
> 
> The current documentation already contains information on this. I think it will be a bit confusing if I include other headers with information about these sensors as they are quite straightforward?
> 
But it does not say anything about battery status, and the other added sensors. I don't see this here.

**Comment by LennP** [comment]
> > > Could you add the types of sensors that are added and describe the configuration details? For example, like they did here:
> > > https://github.com/home-assistant/home-assistant.io/blob/22c2bfe7b276d468f5761ecdac68ec05f8fd78ca/source/_integrations/traccar_server.markdown?plain=1#L120-L137
> > 
> > 
> > The current documentation already contains information on this. I think it will be a bit confusing if I include other headers with information about these sensors as they are quite straightforward?
> 
> But it does not say anything about battery status, and the other added sensors. I don't see this here.

Sure, I added information on the battery sensor, calibration sensor, connection sensor and signal strength sensor

**Review by c0ffeeca7 — DISMISSED** [review]
Nice! Thank you, @LennP :+1:

**Comment by LennP** [comment]
Fixed merge conflicts

**Review by c0ffeeca7 — DISMISSED** [review]

**Review by joostlek — DISMISSED** [review]
There's a merge conflict, can you take a look?

**Comment by LennP** [comment]
> There's a merge conflict, can you take a look?

Done!

File: source/_integrations/motionblinds_ble.markdown [hunk 1]
```diff
@@ -46,6 +47,11 @@ The following entities are available for a Motionblinds Bluetooth device:
   -  Favorite button: allows you to move the blind to the favorite position.
 - [Select](https://www.home-assistant.io/integrations/select/) entities:
   -  Speed select: allows you to change the speed of the motor to low, medium, or high. Available for all blinds except curtain blinds and vertical blinds.
+- [Sensor](https://www.home-assistant.io/integrations/sensor/) entities:
+  -  Battery sensor: shows the battery percentage. The icon also reflects whether the motor is currently charging and/or whether the motor is wired and therefore does not have a battery.
+  -  Calibration sensor: shows whether or not the blind is still calibrated. The motor can move to an uncalibrated state when the motor has been moved to a different position while not powered. Available for a curtain blind and vertical blind as these can be moved while not powered.
+  -  Connection sensor: shows whether the blind is connected, disconnected, connecting, or disconnecting.
+  -  Signal strength sensor: shows the signal strength in dBm.
```
Comment (frenck) [inline source/_integrations/motionblinds_ble.markdown]:
```suggestion
- [Sensor](https://www.home-assistant.io/integrations/sensor/) entities:
  -  Battery sensor: shows the battery percentage. The icon also reflects whether the motor is currently charging and/or whether the motor is wired and, therefore, does not have a battery.
  Calibration sensor: shows whether or not the blind is still calibrated. The motor can move to an uncalibrated state when it has been moved to a different position while not powered. This sensor is available for curtain blinds and vertical blinds, as these can be moved while not powered.
  -  Connection sensor: shows whether the blind is connected, disconnected, connecting, or disconnecting.
  -  Signal strength sensor: shows the signal strength in dBm.
```

**Review by frenck — APPROVED** [review]

**Pull request merged**